README.md

```diff
@@ -35,6 +36,56 @@ $ swaggerhub configure
 ```
 
 Environment variables take precedence over the configuration file created by this command.
+
+## Additional configuration for SwaggerHub On-Premise
+
+If your SwaggerHub On-Premise instance uses a **self-signed or privately signed SSL certificate**, there are additional steps required to make SwaggerHub CLI trust this certificate.
+
+By default, Node.js rejects self-signed or privately signed SSL certificates because their root CA is not known. You will see an error like this in CLI output:
+
+```
+FetchError: request to https://... failed, reason: self signed certificate
+```
+
+The solution is to use the [`NODE_EXTRA_CA_CERTS`](https://nodejs.org/api/cli.html#cli_node_extra_ca_certs_file) environment variable to specify custom trusted certificates for Node.js.
+
+Start by creating a .pem file containing your CA root and intermediate certificates in the PEM format. Certificates can be listed in any order. A self-signed certificate is its own CA, in this case include just the certificate itself. Blank lines are allowed, but optional, between individual certificates.
```

Comment:
This says "Certificates can be listed in any order.", but the SwaggerHub docs say "The order of certificates in the PEM file must be such that each one certifies the one before it." https://app.swaggerhub.com/help/enterprise/config/ssl/certificate

Reply:
Never mind :)